cli/asc.json

```diff
@@ -138,7 +138,12 @@
     "default": 0
   },
   "importTable": {
-    "description": "Imports the function table instance provided by the embedder.",
+    "description": "Imports the function table provided by the embedder.",
+    "type": "b",
+    "default": false
+  },
+  "exportTable": {
+    "description": "Exports the function table.",
```

Comment:
Perhaps mention that it will be at the top level namespace and the other imports are in the `env` namespace.

Reply:
Exports don't have namespaces, like imports, but I can mention that it becomes exported as `table`. Is that what you meant?

Reply:
I thought that you could do the following: `@external("env", "foo")`
So that the instance will have the namespace `env`.  I didn't mean the typescript namespace, rather just `instance.env.table` and `imports.env.table`.

Reply:
Yeah, one can do that on imports, which have a module name and a name, but exports just have a name without nesting. As such, there's no `exports.env.table` by design, only our mangling like `exports["env.table"]` as the loader understands it.